lib/sisimai/bite/email/sendgrid.rb

```diff
@@ -0,0 +1,223 @@
+module Sisimai::Bite::Email
+  module SendGrid
+    # Sisimai::Bite::Email::SendGrid parses a bounce email which created by
+    # SendGrid. Methods in the module are called from only Sisimai::Message.
+    class << self
+    # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/SendGrid.pm
```

Comment:
__[rubocop]__
* Layout/CommentIndentation: Incorrect indentation detected (column 4 instead of 6).

<!-- Comment by [SideCI](https://sideci.com) -->